li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/common/LiKafkaCommonClientConfigs.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License").  See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.clients.common;
+
+
+/**
+ * The common configurations for both LiKafkaProducer and LiKafkaConsumer
+ */
+public final class LiKafkaCommonClientConfigs {
```

Comment:
i think our extensions should start with "lnkd" - so "lnkd.metadata.service.client". that way there's very little chance of a future collision, and its very clear which configs are extensions

Reply:
But I thought we want to be generic here? Like we can have a client for a non-LinkedIn metadata service?

Reply:
the client can be an arbitrary implementation, but the property name that sets the client would start with "lnkd" so as not to be confused (or clash) with vanilla kafka client configs

Reply:
Misunderstood your original comment. As discussed in person, we'll use prefix "li." for new li-apache-kafka-clients specific property names.